core/webapp/vis/src/internal/D3Renderer.js

```diff
@@ -2165,38 +2220,40 @@ LABKEY.vis.internal.D3Renderer = function(plot) {
         }
     };
 
-    var renderErrorBar = function(layer, plot, geom, data) {
-        var colorAcc, sizeAcc, topFn, bottomFn, verticalFn, selection, newBars;
+    var renderErrorBar = function(layer, plot, geom, data, xAcc) {
+        var colorAcc, topFn, bottomFn, verticalFn, selection, newBars;
+        var errorLineWidth = geom.errorWidth != null ? geom.errorWidth : geom.width;
+        var xAcc_ = xAcc || function(row) {return geom.getX(row);};
```

Comment:
Remove the `var` line that defines a bunch of variables, and use const to declare `errorLineWidth` and `xAcc_`. Update the variables below to all be `const =` e.g. `const topFn =`, `const bottomFn =`, etc.
```suggestion
        const errorLineWidth = geom.errorWidth != null ? geom.errorWidth : geom.width;
        const xAcc_ = xAcc ?? row => geom.getX(row);
```

Reply:
done